p4-16/psa/PSA.mdk

```diff
@@ -913,16 +897,15 @@ the contents of several metadata fields in the struct
         drop the packet
         return;   // Do not continue below.
     }
-    if (ostd.egress_port == PORT_RECIRCULATE) {
-        recirculate the packet, i.e. it will go back to starting with the
-            ingress parser;
-        return;   // Do not continue below.
-    }
-
```

Comment:
The code above was not deleted, just moved below the next comment, because I also corrected the occurrence of 'ostd.egress_port' in the original with what it should have been 'istd.egress_port'.